lib/src/model.dart

```diff
@@ -5887,8 +5979,8 @@ abstract class SourceCodeMixin implements Documentable {
 
   String get _crossdartPath {
     var node = element.computeNode();
-    if (node is Declaration && node.element != null) {
-      var source = node.element.source;
+    if (node is Declaration && node.declaredElement != null) {
```

Comment:
curious, what motivated this change?

Reply:
Oh, because element is deprecated, and declaredElement is the replacement.